.github/workflows/python-publish.yml

```diff
@@ -1,36 +1,37 @@
-# This workflows will upload a Python Package using Twine when a release is created
+# This workflow will
+# - build distribution package, pure python wheel
+# - deploy produced distribution package to PyIP
+#
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
-name: Upload Python Package
+name: Deploy Python Package
```

Comment:
I guess `Deploy` might not be the right term, but also `Upload` is not fully describing the action.
Maybe `Publish`, after the action name?

Reply:
changed